blockchain/reactor_test.go

```diff
@@ -157,19 +157,20 @@ func makeBlock(height int64, state sm.State) *types.Block {
 
 // The Test peer
 type bcrTestPeer struct {
-	cmn.Service
+	*cmn.BaseService
```

Comment:
Should just be cmn.BaseService.  bcrTestPeer is used as a pointer so no need for an extra pointer // extra garbage collection overhead.